src/Symfony/Component/Console/Helper/OutputWrapper.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Helper;
+
+/**
+ * Simple output wrapper for "tagged outputs" instead of wordwrap(). This solution is based on a StackOverflow
+ * answer: https://stackoverflow.com/a/20434776/1476819 from SLN.
+ *
+ *  (?:
+ *       # -- Words/Characters
+ *       (                       # (1 start)
+ *            (?>                     # Atomic Group - Match words with valid breaks
+ *                 .{1,16}                 #  1-N characters
+ *                                         #  Followed by one of 4 prioritized, non-linebreak whitespace
+ *                 (?:                     #  break types:
+ *                      (?<= [^\S\r\n] )        # 1. - Behind a non-linebreak whitespace
+ *                      [^\S\r\n]?              #      ( optionally accept an extra non-linebreak whitespace )
+ *                   |  (?= \r? \n )            # 2. - Ahead a linebreak
+ *                   |  $                       # 3. - EOS
+ *                   |  [^\S\r\n]               # 4. - Accept an extra non-linebreak whitespace
+ *                 )
+ *            )                       # End atomic group
+ *         |
+ *            .{1,16}                 # No valid word breaks, just break on the N'th character
+ *       )                       # (1 end)
+ *       (?: \r? \n )?           # Optional linebreak after Words/Characters
+ *    |
+ *       # -- Or, Linebreak
+ *       (?: \r? \n | $ )        # Stand alone linebreak or at EOS
+ *  )
+ *
+ * @author Krisztián Ferenczi <ferenczi.krisztian@gmail.com>
+ *
+ * @see https://stackoverflow.com/a/20434776/1476819
+ */
+class OutputWrapper implements OutputWrapperInterface
```

Comment:
is this class really meant to be this open? Almost everything is non-final & public... With the current implementation, I don't think you would really benefit from inheriting the class instead of decorating it so I would suggest to mark it as final instead